### PR TITLE
Updated send-a-message page to V6

### PR DESCRIPTION
**Pull request by SzymonPobiega**
@Particular/tf-asyncawait plase review

### CODE REVIEW

File: Snippets/Snippets_3/BasicUsageOfIBus.cs [hunk 1]
```diff
@@ -6,13 +6,31 @@ public class BasicUsageOfIBus
     {
         void Send()
         {
-            IBus bus = null;
-
             #region BasicSend
+            IBus bus = CreateAndStartTheBus();
```
Comment (timbussmann) [inline Snippets/Snippets_3/BasicUsageOfIBus.cs]: in the v5 example you used the real config api to create the IBus instance which I like much better. Any reason to not do the same in v3/4?

File: nservicebus/messaging/send-a-message.md [hunk 2]
```diff
@@ -6,12 +6,17 @@ redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-To send a message, use the `Send` method on the `IBus` interface, passing as the argument the instance of the message to deliver:
+Prior to V6 sending a message involved using the `Send` method on the `IBus` interface, passing as the argument the instance of the message to deliver. In V6 the equivalent of the `IBus` is the `IBusContext` family of interfaces.
```
Comment (timbussmann) [inline nservicebus/messaging/send-a-message.md]: I somehow don't like that we start of with version difference. I don't think the renaming of interfaces changes much in the scope of sending a basic message. So could we put the interface name and version a little bit in the background? Something like "[...] using the `Send` method on the `IBus`(pre v6)/`IBusContext`(v6) interface[...]". Maybe skip the version altogether an let the samples speak for themselves?

Comment (SeanFeldman) [inline nservicebus/messaging/send-a-message.md]: @timbussmann like this: "Message sending involves using `Send` operation that take an argument of the message to be delivered. Refer to samples for details." ?

**Comment by SzymonPobiega** [comment]
@timbussmann @SeanFeldman I've updated according to your suggestions.

**Comment by SimonCropp** [comment]
thanks @SzymonPobiega

**Pull request merged**